### ISSUE TITLE
💬 [Question] Recommending FireFox As Best For HC.social?

### ISSUE THREAD

**Issue by Tirokk** [Human-Connection/Human-Connection#2873]
## :speech_balloon: Question

@datenbrei decided to recommend FireFox as the best for using `hc.social`!
See 🐛 [Bug] Scrolling fails after a while #2253 on all Chromium platforms!
https://github.com/Human-Connection/Human-Connection/issues/2253#issuecomment-576654226

@alina-beck stated we could do this even on the start page. And I ( @Tirokk ) do agree.

**Comment by sushidave** [comment]
Hold on, please. I noticed the same issue on the latest Firefox version. The bug appeared when scrolling down. It happened at posts around 15 to 20 days before the current date.

**Comment by Tirokk** [comment]
I was never able to see this error on FireFox and Safari on macOS.
Can you double check, please, if this error occurs really on your FireFox again and put a picture here @sushidave .

**Comment by sushidave** [comment]
Yes, I just double checked. This time, Firefox was able to display old posts (from a month ago) without any issue #2253 . So for me, promoting Firefox is good to go!

**Comment by Tirokk** [comment]
Okay, that’s my guess  and experience.

So @sushidave  asked on Zoom if Chrome and Opera should not be suggested in the FAQs, but only FireFox for now @datenbrei ?

**Comment by datenbrei** [comment]
Let's recommend Firefox and tell, that other browsers may function or not. We should not go too deep into technical details for the usual user.

**Comment by alina-beck** [comment]
Yes, I agree! And maybe we could add a section in the FAQs – "Why does Human Connection recommend Firefox?"... I could help with writing an explanation that includes the technical details while still being understandable to regular users, if you want. :wink:

**Comment by datenbrei** [comment]
Yes, I want. ;-)

I have to beware of not writing too bad about other brothers, although they are some kind of espionage machines.

**Comment by alina-beck** [comment]
@sushidave is working on adding this to the FAQs – do we want to also notify users (who are not using Firefox) about our recommendation when they visit our page?

I think we could:
A) Add an extra line to the `login` and/or `sign up` form: "HC.social runs best on Firefox" (could be annoying for Firefox users)
B) Add this extra line _only_ when users are not on Firefox already (more work, because we need to detect the used browser somehow)
C) Trust people to file bug reports and visit the FAQs when things are not working as expected :wink:

What do you think @Human-Connection/hc-dev-team?

**Comment by sushidave** [comment]
Publish as an FAQ - done.

I recommend to add it just to the sign-up form and only if the browser is not Firefox - or it would be annoying.

**Comment by Tirokk** [comment]
Is not stale!
